### ISSUE TITLE
Getting in Browser console "Unknown property ‘-moz-border-radius-bottomright’.  Declaration dropped"

### ISSUE THREAD

**Issue by epugh** [MiniProfiler/rack-mini-profiler#504]
I believe that in more recent versions of Firefox, we don't need these variations that are browser specific.

Here is what is in my Developer Console.  I am on version 2.3.2 as well.  

```
Unknown property ‘-moz-border-radius-bottomright’.  Declaration dropped. includes.css:237:40
Elements matching selector: .profiler-results.profiler-top.profiler-left.profiler-no-controls .profiler-totals, .profiler-results.profiler-top.profiler-left.profiler-no-controls .profiler-result:last-child .profiler-button, .profiler-results.profiler-top.profiler-left .profiler-controls
NodeList [ div.profiler-button.profiler-totals, div.profiler-button ]
```

Am I the only one who gets these warnings?  Is it specific to rack-mini-profiler or is it maybe something in my Rails app integration of this gem?

**Comment by SamSaffron** [comment]
Oh ... -moz-border-radius` is such a dinosaur of a property. Feel free to send through a PR to drop it.

**Comment by epugh** [comment]
Please see #505 for a PR that does this!

**Comment by nateberkopec** [comment]
Closed by #505
